portfolio/src/main/webapp/scripts/script.js

```diff
@@ -72,4 +72,13 @@ const revealSecretMessage = () => {
 const playAudio = (audioClip) => {
 	const audio = new Audio(audioClip);
 	audio.play();
+}
+
+/*
+ * Fetches data from java servlet and adds it to DOM tree
+ */
+const fetchAndInsertDataIntoDOM = async () => {
```

Comment:
It looks like this currently works the way you have it, but as a general piece of advice you might want to consider this-- I think, unless there's a very well-defined scope for your functions, you probably should be in the practice of declaring them as functions and not as const because in case you end up using them elsewhere (e.g. as helpers in other functions), you'll start having to worry about their ordering here else run into undefined issues.

For reference: https://stackoverflow.com/questions/33040703/proper-use-of-const-for-defining-functions-in-javascript

Reply:
Nice point, something I hadn't considered before!

Reply:
That's a good point Colette! I've tended to use arrow functions for all my projects in the past and haven't come across this issue, but it makes sense why explicit function declarations might be easier to deal with when the complexity of the script increases. That being said, I do still think there are intrinsic benefits of using arrow functions over function declarations as also pointed out in that SO post you linked, and that when possible, it might not be a bad idea to keep them in.